src/loadFileConfigs.ts

```diff
@@ -17,9 +30,4 @@ export type Configurations<T> = {[key in keyof T]: any}
  */
 export const loadFileConfigs = <T extends NonConfigEnv>(
   process: T
-): Configurations<ConfigTypes> => {
-  const itar: any = R.mapObjIndexed(
-    R.ifElse(fs.existsSync, require, R.always({}))
-  )
-  return itar(configPaths(process))
-}
+): Configurations<ConfigTypes> => readConfigFiles(configPaths(process))
```

Comment:
this change seems unnecessary. Can you confirm?
If not needed can you revert this file.